.github/ISSUE_TEMPLATE

```diff
@@ -0,0 +1,18 @@
+Hi there!
+
+Thanks for submitting an issue to Protractor.
+
+To help us help you better, please do the following before submitting an issue:
+
+1. Review the questions section of [CONTRIBUTING.md](https://github.com/angular/protractor/blob/master/CONTRIBUTING.md#contributing)
+2. Ensure that your issue is a bug or a feature request, and not a usage or debugging question
+3. Fill in the information below:
+4. Delete this intro text :smile:
```

Comment:
Would it be terrible to add something like "If you don't delete the intro text, we'll assume you didn't read the instructions and auto-close your issue"?